flink-tests/src/test/java/org/apache/flink/test/checkpointing/RegionFailoverITCase.java

```diff
@@ -110,15 +117,33 @@
 	@ClassRule
 	public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
 
+	private final String schedulerType;
+
+	public RegionFailoverITCase(final String schedulerType) {
+		this.schedulerType = checkNotNull(schedulerType);
+	}
+
+	@Parameterized.Parameters
```

Comment:
```suggestion
	@Parameterized.Parameters(name = "scheduler = {0}")
```
to make the test more identifiable.

Reply:
Good suggestion.